packages/assets-controllers/src/multi-chain-accounts-service/multi-chain-accounts.ts

```diff
@@ -0,0 +1,47 @@
+import { handleFetch } from '@metamask/controller-utils';
+
+import type { GetBalancesQueryParams, GetBalancesResponse } from './types';
+
+const MULTICHAIN_ACCOUNTS_DOMAIN = 'https://accounts.api.cx.metamask.io';
+
+const getBalancesUrl = (address: string) =>
+  `${MULTICHAIN_ACCOUNTS_DOMAIN}/v2/accounts/${address}/balances`;
+
+/**
+ * Fetches Balances for multiple networks.
+ * @param address - address to fetch balances from
+ * @param options - params to pass down for a more refined search
+ * @returns a Balances Response
+ */
+export async function fetchMultiChainBalances(
+  address: string,
+  options?: GetBalancesQueryParams,
```

Comment:
Hmm currently a 1:1 mapping for the GET balances query params. We might make this a bit more friendly to work with. Will see from the integration.

Reply:
It's important to note that by default the balances API will fetch from **multiple networks**
https://docs.cx.metamask.io/api/multichainAccounts/#tag/Balances/operation/AccountsController_getAccountBalancesV2

```
Default: "1,137,56,59144,8453,10,42161,534352"
```